build_packages.py

```diff
@@ -13,6 +14,10 @@
 from src.repository import Repository
 from src.version import FullVersion
 
+# Setup logging
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
```

Comment:
L18 overrides the logging module's default level: warning.
L19 is not necessary right now, I intend to use it very soon